src/rpm/builder.rs

```diff
@@ -0,0 +1,216 @@
+use chrono::Utc;
+use gethostname::gethostname;
+use std::ffi::OsString;
+use std::fs::OpenOptions;
+
+use super::info::RPMInfo;
+
+struct InnerPath {
+    path: String,
+    user: String,
+    group: String,
+    mode: u8,
+}
+
+#[derive(Debug, Default)]
```

Comment:
Default enables bad code like `RPMBuilder::default().build()`.

Reply:
Good point. It was used to fast start. It should be removed by the end of this PR.

Reply:
@andy128k,
According to `builder_derive`, they uses exactly this pattern:  `RPMBuilder::default().something().build()`